mergin/test/test_client.py

```diff
@@ -786,18 +798,27 @@ def test_available_storage_validation(mcStorage):
     try:
         mcStorage.push_project(project_dir)
     except ClientError as e:
+        print("Pushe fail")
         # Expecting "You have reached a data limit" 400 server error msg.
         assert "You have reached a data limit" in str(e)
         got_right_err = True
-    assert got_right_err
+    finally:
+        assert got_right_err
```

Comment:
Shouldn't we check if we for true if we except this error to happen ?

```suggestion
        assert got_right_err==True
```